samples/net/dsa/src/dsa_lldp.c

```diff
@@ -146,21 +211,18 @@ void dsa_lldp_print_info(uint8_t *lldp_p, uint8_t lanid)
 	} while (1);
 }
 
-int dsa_lldp_recv(struct net_if *iface, struct instance_data *pd,
-		  uint16_t *lan, int *origin_port,
-		  struct eth_addr *origin_addr)
+static int dsa_lldp_recv(struct net_if *iface, struct instance_data *pd, uint16_t *lan,
+			 int *origin_port, struct eth_addr *origin_addr)
 {
 	struct ethernet_context *ctx = net_if_l2_data(iface);
-	struct net_eth_hdr *eth_hdr =
-		(struct net_eth_hdr *) pd->recv_buffer;
+	struct net_eth_hdr *eth_hdr = (struct net_eth_hdr *)pd->recv_buffer;
 	uint8_t *lldp_p = &pd->recv_buffer[sizeof(struct net_eth_hdr)];
 	int received;
 
 	*lan = ctx->dsa_port_idx;
 
 	/* Receive data */
-	received = recv(pd->sock, pd->recv_buffer,
-			sizeof(pd->recv_buffer), 0);
+	received = recv(pd->sock, pd->recv_buffer, sizeof(pd->recv_buffer), 0);
```

Comment:
Will it pass the checkpatch ?

Reply:
Yes. I did clang-format when created the file.